src/services/creation/character/character-creation.service.ts

```diff
@@ -1,74 +1,105 @@
-import _ from "lodash"
-import { Message } from "discord.js"
-import { TextFormats } from "../../../enums/text-formats.enum"
-import { DisplayMessageService } from "../../display-message/display-message.service"
-import { CharacterService } from "../../character/character.service"
-import { MessageFormattingService } from "../../message-formating/message-formatting.service"
-import { StateManagerService } from "../../state-manager/state-manager.service"
-import { StateNamesEnum } from "../../../enums/state-names.enum"
+import { Message } from "discord.js";
+import _ from "lodash";
+import { StateNamesEnum } from "../../../enums/state-names.enum";
+import { CharacterService } from "../../character/character.service";
+import { DisplayMessageService } from "../../display-message/display-message.service";
+import { StateManagerService } from "../../state-manager/state-manager.service";
+import { MessageFormattingService } from "../../message-formating/message-formatting.service";
+import { TextFormatsEnum } from "../../../enums/text-formats.enum";
 
 export class CharacterCreationService {
-  private static _instance: CharacterCreationService
+  private static _instance: CharacterCreationService;
 
   public static getInstance(): CharacterCreationService {
-    if(_.isNil(CharacterCreationService._instance)) {
-      CharacterCreationService._instance = new CharacterCreationService()
+    if (_.isNil(CharacterCreationService._instance)) {
+      CharacterCreationService._instance = new CharacterCreationService();
     }
-    return CharacterCreationService._instance
+    return CharacterCreationService._instance;
   }
 
   public init(message: Message): void {
-    CharacterService.getInstance().getEntity(message.author.id)
-    .then((characterFound): void => {
-      if (!characterFound) {
-        const memberId = message.author.id
+    CharacterService.getInstance()
```

Comment:
To enhance your code, returning a promise here could be a good idea.
Just add return and change the type.
Add void as a prefix when calling `init()`

Reply:
Ok. I will add all that.